QuickStart.html

```diff
@@ -7,231 +7,63 @@
-Next, download <a href="http://us2.metamath.org:8888/metamath/set.mm">set.mm</a>
-(6MB) into the directory&nbsp;<code>/Users/</code><span
- style="font-style: italic; font-weight: bold;">userone</span><code>/</code><code>metamath</code>.&nbsp;
-Or,
-if
-you
-want
-it
-somewhere else, change the parameter <br>
-<br>
-<code style="font-weight: bold;">&nbsp; </code><code
- style="font-weight: bold;">LoadFile,set.mm</code><br>
-<br>
-accordingly, using <code style="font-weight: bold;">TextEdit.app</code>
-to edit<br>
-<br>
-<code style="font-weight: bold;">&nbsp;</code><code>
-<span style="font-weight: bold;">/Users/</span></code><span
- style="font-style: italic; font-weight: bold;">userone</span><code
- style="font-weight: bold;">/mmj2/mmj2jar/RunParms.txt<br>
-</code><br>
-You will need to update the mmj2Path and MetamathPath arguments in <a
- href="mmj2jar/MacMMJ2.command"><code style="font-weight: bold;">MacMMJ2.command</code></a>
-(command line
-arguments #3 and 4 after "<code>mmj2.jar</code>" -- see <a
- href="doc/mmj2CommandLineArguments.html">mmj2CommandLineArguments</a>)
-in:<br>
-<br>
-<code style="font-weight: bold;">&nbsp;</code> <code
- style="font-weight: bold;">/Users/</code><span
- style="font-style: italic; font-weight: bold;">userone</span><code><span
- style="font-weight: bold;">/mmj2/mmj2jar/MacMMJ2.command</span><br>
-<br>
-&nbsp;</code>and while you're doing this, update this one too:<code><br>
-</code><br>
-<code style="font-weight: bold;">&nbsp;</code> <code
- style="font-weight: bold;">/Users/</code><span
- style="font-style: italic; font-weight: bold;">userone</span><code><span
- style="font-weight: bold;">/mmj2/mmj2jar/MacPATutorial.command</span><br>
-</code><br>
-<code><br>
-</code>Then&nbsp; double-click the <code>/Users/</code><span
- style="font-style: italic; font-weight: bold;">userone</span><code>/mmj2/mmj2jar/MacMMJ2.command</code>
-file in the Finder window<br>
-<br>
-<br>
-<span style="font-style: italic;"></span>
-<hr style="width: 100%; height: 2px;"><br>
+<h1>Quick Start</h1>
+<p>
+First, install the software
+(See <a href="README..md">README.md</a> and
+<a href="INSTALL.html">INSTALL.html</a>).
+We'll assume you also have the Metamath set.mm database installed;
+by default we assume it's in the default directory
+c:\set.mm\ (Windows) or "$HOME/set.mm/" (everything else).
+We'll also assume that mmj2 is installed in
+C:\mmj2\ (Windows) or "$HOME/mmj2" (everything else).
+Then:
+
+<ul>
+<li>Windows: Double-click \mmj2\mmj2jar\mmj2.bat from Windows Explorer
+<li>MacOS: Double-click mmj2/mmj2jar/MacMMJ2.command from Finder
+<li>Linux/Unix: Double-click mmj2/mmj2jar/mmj2.sh from your GUI shell
+<li>Cygwin: Run mmj2/mmj2jar/mmj2-cygwin.bat
+</ul>
+
+<p>
+These files will try to load the Metamath database set.mm, by default
+from the directory c:\set.mm on Windows and "$HOME/set.mm".
+You can change this, by either setting the environment variable
+METAMATH_DB_DIR to the directory you want to use, or can copy that
```

Comment:
This is the first I've heard of this environment variable. Do command line arguments not work anymore?

Reply:
Command like arguments work for mmj2.jar, just like they always have.

But using mmj2.jar is an absolute pain, so we have scripts that try to hide some of the nonsense, but those scripts have *fixed* file locations. E.g., mmj2.bat and so on. I added the "METAMATH_DB_DIR" to have *some* capability, but mmj2 doesn't have a way to give the database on the command line, so that was the closest I could get to normal.